src/MultiRegion/multi_region_output_writers.jl

```diff
@@ -6,6 +6,8 @@ import Oceananigans.OutputWriters:
                         serializeproperty!
 
 # This is working just fine at the moment?
+# But it will be veeeeery slow, as reconstruct_global_field is not 
+# a performant operation
 function fetch_output(mrf::MultiRegionField, model)
     field = reconstruct_global_field(mrf)
     compute_at!(field, time(model))
```

Comment:
Another possibility is to `@apply_regionally` to `fetch_output` right?

Then maybe we also want to extend `convert_output` to handle a `MultiRegionObject` or something (eg just save as a tuple, converting each element).